src/pages/index.md

```diff
@@ -82,15 +82,15 @@ You can now integrate high performance APIs into any application with ease using
 
 <TextBlock slots="image, heading, text, buttons" theme="light" />
 
-![exc-banner](./images/F_Illu_DevEcoFirefly_discovery_banner_746x500_2x.png)
+![cc-banner](./images/F_Illu_DevEcoCCdiscovery_banner_746x500_2x.png)
 
-### Build Cloud Native Adobe Apps
+### Build for Creative Cloud
 
-Project Firefly is a complete framework to build and deploy custom web apps on Adobe I/O Runtime, our serverless platform. Leverage React Spectrum (Adobe’s UI toolkit), create microservices, fire custom events, and orchestrate APIs to create custom apps for your business users.
+The Creative Cloud Developer Platform is a collection of APIs and SDKs that let you extend and integrate with Creative Cloud apps and services, which are used by millions of people around the world. From automating workflows to integrating your software with Creative Cloud, our developer platform has the tools you need to power creativity.
 
-* [Learn more](/apis/experienceplatform/project-firefly.html)
+* [Learn more](https://www.adobe.io/creative-cloud/)
```

Comment:
@pathyde1126 pls use relative paths, in this case `/creative-cloud/` is enough, there's no need to specify the hostname (especially since we're moving away from it :D )